azure-bootstrap.yml

```
@@ -0,0 +1,45 @@
+  autoCancel: true
+
+# add nf-tools repo to resources (for Azure Pipelines templates)
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: nanoframework/nf-tools
+      endpoint: nanoframework
+
+jobs:
+- job: Trigger
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)